TODO.md

```diff
@@ -68,3 +68,5 @@
   every read/write range command.
 
 * Gossip should be changed to send only protobufs.
+* persist Gossip and the kv.leaderCache to make sure that restarting
+  a node is harmless.
```

Comment:
The leader cache doesn't the option of prefetching anything, so you will have to query each range to learn about its leader. Performance wise it should be fine for the leader cache, you could simply persist on update, and never touch it again unless the node is actually in the process of booting.
@spencerkimball thought it was a worth persisting Gossip as well.
I don't have strong opinions either way.